cases.json

```diff
@@ -24,83 +24,120 @@
             }
         },
         {
-            "name" : "Connect-Compatible_Ciphers_1",
-            "client" : {
-                "min_version" : 770,
-                "max_version" : 772,
-                "cipher" : "TLS_AES_128_GCM_SHA256"
+            "name" : "Connect-TLS13-Ciphers",
+            "shared_params" : {
+              "ciphers" : [
+                "TLS_AES_128_GCM_SHA256;AES128-GCM-SHA256",
+                "TLS_CHACHA20_POLY1305_SHA256;CHACHA20-POLY1305-SHA256",
+                "TLS_AES_256_GCM_SHA384;AES256-GCM-SHA384"
+              ]
             },
-            "server" : {
-                "min_version" : 770,
-                "max_version" : 772,
-                "cipher" : "TLS_AES_128_GCM_SHA256"
-            }
-        },
-        {
-            "name" : "Connect-Compatible_Ciphers_3",
             "client" : {
                 "min_version" : 770,
-                "max_version" : 772,
-                "cipher" : "TLS_AES_256_GCM_SHA384"
+                "max_version" : 772
```

Comment:
So this enables all TLS versions but 1.3 should get picked. Shall we test only 1.3 enabled as well? When we can handle negative tests we could also test 1.3 not enabled and 1.2 cipher suites with only 1.3 enabled.

Reply:
That makes sense, I will adjust that.